packages/m2m-gateway/test/api/tenants/addCertifiedAttribute.test.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { describe, it, expect, vi } from "vitest";
+import { generateId } from "pagopa-interop-models";
+import { generateToken } from "pagopa-interop-commons-test";
+import { AuthRole, authRole } from "pagopa-interop-commons";
+import request from "supertest";
+import { api, mockTenantService } from "../../vitest.api.setup.js";
+import { appBasePath } from "../../../src/config/appBasePath.js";
+import {
+  missingMetadata,
+  resourcePollingTimeout,
+} from "../../../src/model/errors.js";
+
+describe("POST /tenants/:tenantId/certifiedAttributes router test", () => {
+  const makeRequest = async (token: string, body: Record<string, unknown>) =>
```

Comment:
```suggestion
  const makeRequest = async (token: string, body: m2mGatewayApi.TenantCertifiedAttributeSeed) =>
```
We can use the right type and cast where it's necessary as done [here](https://github.com/pagopa/interop-be-monorepo/pull/1801#discussion_r2084856154)